nexus/Cargo.lock

```diff
@@ -435,6 +435,26 @@ dependencies = [
  "tracing",
 ]
 
+[[package]]
```

Comment:
this update has aws sdk getting split on two versions of `aws-smithy-http` let's wait for that to converge